runtime/compiler/runtime/J9Profiler.hpp

```diff
@@ -174,8 +177,16 @@ class TR_PersistentProfileInfo
 
    void dumpInfo(TR::FILE *);
 
-   private:
+   void setCallSiteInfo(TR::Compilation *comp, uint32_t numCallSites, TR_InlinedCallSite *inlinedCallSite);
```

Comment:
Making this public seems suboptimal - the isolation provided by the previous private declaration helps to prevent accidental corruption of the profiling data. Is there an overall design for this work that explains this design choice?